TEST_APPS/testcases/nfc/README.md

```diff
@@ -0,0 +1,319 @@
+     +-------+     +-------+             +--------+                  
+     | [NC]  |     | [B8]  |             |[ 2][ 1]|                  
+     | [IOREF|     | [B9]  |             |[ 4][ 3]|                  
+     | [RST] |     | [AVDD]|             |[ 6][ 5]|                  
+1<---+ [3V3] |     | [GND] |             |[ 8][ 7]|                  
+     | [5V]  |     | [A5]  +--->23       |[10][ 9]|                  
+     | [GND] |     | [A6]  +--->21       |[12][11]|                  
+25<--+ [GND] |     | [A7]  +--->19       |[14][13]|                  
+     | [VIN] |     | [B6]  +--->3        |[16][15]|                  
+     |       |     | [C7]  |             |[18][17]|                  
+26<--+ [A0]  |     | [A9]  |             |[20][19]|                  
+16<--+ [A1]  |     | [A9]  |             |[22][21]|                  
+     | ...   |     |       |             |[24][23]|                  
+     |       |     | [A8]  |             |[26][25]|                  
+     +-------+     | ...   |             +--------+                  
+                   |       |                               
+                   |       |                               
+                   +-------+                               
+                                         
+Patch using jumper wires to the             
+indicated pins on the Shield.            
+                                         
+```
+Schematic (https://www.element14.com/community/docs/DOC-76384/l/explore-nfc-board-schematic)
+To change pinouts, if your reference design or shield pins differ for the PN512 controller driver, open nfcProcessCtrl.cpp and find the code
+```
+NFCProcessController::NFCProcessController(events::EventQueue &queue) :
+        // pins: mosi, miso, sclk, ssel, irq, rst
+        _pn512_transport(D11, D12, D13, D10, A1, A0), _pn512_driver(
+                &_pn512_transport), _queue(queue), _nfc_controller(
+                &_pn512_driver, &queue, _ndef_buffer) {
+}
+```
+modify pins as needed.
+
+**Compilation target drivers**
+
+If using the EEPROM driver, the mbed_app.json will contain
+```
+    "target_overrides": {
+        "DISCO_L475VG_IOT01A": {
+            "target.extra_labels_add": ["M24SR"]
+        }
+    }
+...
+```
+If using the Explorer Shield or PN512 driver mbed_app.json will add
+```
+    "target_overrides": {
+        "NUCLEO_F401RE": {
+            "target.extra_labels_add": ["PN512"]
+        }
+    }
+...
+```
+## Alternate NFC drivers note:
+
+Please see the example json file .\TEST_APPS\testcases\nfc\mbed_app.json . The test does not check that you have any needed shield installed, so if it "hangs" at the point the "initnfc" command is used, the driver or shield may be the fault. The test assumes that MBED_CONF_NFCEEPROM is set to 1, if not it assumes that a controller driver is in use. To add support for other then PN512 and M24SR, it is required to add the support. 
```

Comment:
"To add support for other then PN512 and M24SR, it is required to add the support. "
that is true but somewhat tautological

also then -> than